CHANGELOG.md

```diff
@@ -1,3 +1,39 @@
+<a name="2.0.0-rc.12"></a>
+
+# [2.0.0-rc.12](https://github.com/commercetools/ui-kit/compare/2.0.0-rc.11...2.0.0) (2018-11-23)
+
+## BREAKING CHANGES
+
+- `MoneyInput`: dropped support for `hasAmountError`, `hasAmountWarning`, `hasCurrencyError` and `hasCurrencyWarning`. Use `hasError` and `hasWarning` instead. ([9e00148](https://github.com/commercetools/ui-kit/commit/9e00148)), closes [#175](https://github.com/commercetools/ui-kit/issues/175)
+- `MoneyInput`: `isTouched` only returns true when both fields were touched from now on. ([9e00148](https://github.com/commercetools/ui-kit/commit/9e00148)), closes [#175](https://github.com/commercetools/ui-kit/issues/175)
+- `media-queries`: need to be imported differently now [#218](https://github.com/commercetools/ui-kit/pull/218)
+
+* You need to explicitly add [our peer deep deps](https://github.com/commercetools/ui-kit/blob/20a9315/package.json#L210-L220) ([#156](https://github.com/commercetools/ui-kit/issues/156)) ([ae59216](https://github.com/commercetools/ui-kit/commit/ae59216))
+* We dropped some images which were hosted in ui-kit but didn't belong here ([#230](https://github.com/commercetools/ui-kit/issues/230)) ([3f3dadf](https://github.com/commercetools/ui-kit/commit/3f3dadf))
+* Consumers of ui-kit no longer need to `import '../materials/internals/reset.mod.css';` and `import '../materials/internals/grid.mod.css';` ([#186](https://github.com/commercetools/ui-kit/issues/186)) ([7a9e27b](https://github.com/commercetools/ui-kit/commit/7a9e27b))
+
+## Bug Fixes
+
+- `SelectInput`, `CreatableSelectInput`, `AsyncSelectInput`, `AsyncCreatableSelectInput`: align indicator icons properly ([#190](https://github.com/commercetools/ui-kit/issues/190)) ([b2f0141](https://github.com/commercetools/ui-kit/commit/b2f0141))
+- `MoneyInput`: Avoid rounding errors ([#229](https://github.com/commercetools/ui-kit/issues/229)) ([b92ede4](https://github.com/commercetools/ui-kit/commit/b92ede4))
+
+* `MoneyInput`: fix group style ([#188](https://github.com/commercetools/ui-kit/issues/188)) ([68430ab](https://github.com/commercetools/ui-kit/commit/68430ab))
+
+- `LocalizedTextField`: expose `LocalizedTextField` form main exports ([#233](https://github.com/commercetools/ui-kit/issues/233)) ([f04256d](https://github.com/commercetools/ui-kit/commit/f04256d))
+- styles: Remove `overflow: hidden` from body style ([#189](https://github.com/commercetools/ui-kit/issues/189)) ([a6b4b3b](https://github.com/commercetools/ui-kit/commit/a6b4b3b))
+- replace styled-components with react-emotion ([#184](https://github.com/commercetools/ui-kit/issues/184)) ([869a036](https://github.com/commercetools/ui-kit/commit/869a036))
+
+## Features
+
+- `ConnectorIcon`: added `ConnectorIcon`([#243](https://github.com/commercetools/ui-kit/issues/243)) ([36ef073](https://github.com/commercetools/ui-kit/commit/36ef073))
```

Comment:
I think we can remove this, it's already mentioned below `add several new icons`